DRAFT.md

```diff
@@ -743,41 +757,55 @@ explicit constexpr indirect(allocator_arg_t, const Allocator& a);
 10. _Throws_: Nothing unless `allocator_traits<allocator_type>::allocate` or
   `allocator_traits<allocator_type>::construct` throws.
 
+```c++
+template <class U>
+explicit constexpr indirect(U&& u);
+```
+
+11. _Constraints_: `is_constructible_v<T, U&&>` is `true`.
```

Comment:
I think this constraint is wrong, as this will result in a difference between polymorphic and indirect behaviour.  This should restrict U to be `is_same<T, remove_cvref<U>>`